grobid-core/src/main/java/org/grobid/core/data/BiblioItem.java

```diff
@@ -1990,8 +1990,16 @@ public String toBibTeX(String id, GrobidAnalysisConfig config) {
                 if (isoDate != null) {
                     bibtex.add("  date = {" + isoDate + "}");
                 }
-            }
-            if (publication_date != null) {
+                if (normalized_publication_date.getYear() >= 0) {
+                    bibtex.add("  year = {" + normalized_publication_date.getYear() + "}");
+                }
+                if (normalized_publication_date.getMonth() >= 0) {
+                    bibtex.add("  month = {" + normalized_publication_date.getMonth() + "}");
+                }
+                if (normalized_publication_date.getMonth() >= 0) {
```

Comment:
it should be `normalized_publication_date.getDay()` here, corrected when added to #807